flink-libraries/flink-table/src/main/scala/org/apache/flink/table/runtime/functions/ScalarFunctions.scala

```diff
@@ -266,6 +268,21 @@ object ScalarFunctions {
     regexp_extract(str, regex, 0)
   }
 
+  /**
+    * Returns the ASCII code value of the leftmost character of the string str.
+    */
+  def ascii(str: String): Integer = {
+    if (str == null || str.equals("")) {
+      0
+    } else {
+      if (CharMatcher.ASCII.matches(str.charAt(0))) {
+        str.charAt(0).toByte.toInt
+      } else {
+        0
```

Comment:
Hi @xccui I tried to change the value of this method that originally returned 0 to null, but both related tests throw NPE, I don't know if it is related to Integer. Can you try it?

Reply:
@xccui sorry for sending many duplicated comment, Github service is not available yesterday.

Reply:
@xccui